src/pages/Home/_MapPage.scss

```diff
@@ -249,6 +246,20 @@
       color: #dee2e6;
     }
   }
+  .sc-controls-left {
+    @media screen and (max-width: 900px) {
+      display: none;
+    }
+  }
+}
+
+#map-readonly {
+  .continent-container {
+    transform: translate(0, 44px);
```

Comment:
For native CSS styling, please consider using https://autoprefixer.github.io/ (set browserlist config to last 1 version) to add vendor prefixes, also for here and elsewhere :)